README.md

```diff
@@ -283,10 +283,10 @@ mOcKiNgCaSe can be used in a node environment, as well as in the browser. You ca
 </script>
 </html>
 ```
-### Self Hosting
+### Self Hosting [:arrow_up:](#browser-usage)
```

Comment:
As **Self Hosting** and **CDN Usage** are small sections, I don't see a point to have a backlink to **Browser Usage**.

But **Browser Usage** should have a backlink to `#api` as it's a part of the table of contents

Reply:
Ah gotcha! I misunderstood the original request to add link(s) ti the Browser Usage section as linking its children instead of also linking it to API. Updated 🍻